package-lock.json

```diff
@@ -3352,7 +3365,8 @@
     "caniuse-db": {
       "version": "1.0.30001084",
       "resolved": "https://registry.npmjs.org/caniuse-db/-/caniuse-db-1.0.30001084.tgz",
-      "integrity": "sha512-oxprDntKFaMkhNV0vfFtaKtGDAJda8rdCsWC//6WTXsJgb44NxTImwkYym1uFiKkawI0SsC/y3Lj8gxJjH3w3A=="
+      "integrity": "sha512-oxprDntKFaMkhNV0vfFtaKtGDAJda8rdCsWC//6WTXsJgb44NxTImwkYym1uFiKkawI0SsC/y3Lj8gxJjH3w3A==",
+      "dev": true
```

Comment:
If I understand correctly, `"dev": true` means a dependency will not be included in a `production` build. 

Why are all of these dependencies now `"dev": true`? Should they be?

(Very happy if we are actually able to remove these from a prod build though!)

Reply:
@ivan-c You are right.  Looking over those dependencies that are now marked as `"dev":"true"` are indeed development dependencies only.  The file is automatically re-generated by npm.  It could be that they are now marked as such after I updated an dependency and that in turn updated the node_module tree